stable/rabbitmq/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: rabbitmq
-version: 5.1.0
+version: 5.2.0
```

Comment:
```suggestion
version: 5.3.0
```

Reply:
https://github.com/helm/charts/blob/master/stable/rabbitmq/Chart.yaml#L3

Reply:
Ok, i updated the version.